contracts/VToken.sol

```diff
@@ -1226,8 +1257,12 @@ contract VToken is
      * @notice Reduces reserves by transferring to the protocol reserve contract
      * @dev Requires fresh interest accrual
      * @param reduceAmount Amount of reduction to reserves
+     * @param kind kind of reserve
      */
-    function _reduceReservesFresh(uint256 reduceAmount) internal {
+    function _reduceReservesFresh(uint256 reduceAmount, IProtocolShareReserve.IncomeType kind) internal {
```

Comment:
`kind` is always `IProtocolShareReserve.IncomeType.SPREAD`. I would remove the param and I would hardcode the enum value in this internal function.